packages/components/organisms/f-checkout/src/components/_tests/AgeVerification.test.js

```diff
@@ -31,19 +31,111 @@ describe('AgeVerification', () => {
 
     describe('computed ::', () => {
         describe('days ::', () => {
-            beforeEach(() => {
-                // Arrange
-                wrapper = shallowMount(AgeVerification, {
-                    i18n,
-                    localVue,
-                    store: createStore()
+            describe('when selected month should has 31 days', () => {
```

Comment:
I would probably combine all of these `describes` into one, with the expected number of days as a third parameter